api/state.go

```diff
@@ -16,6 +16,7 @@ type StateAPI struct {
 }
 
 func (state *StateAPI) Get(w http.ResponseWriter, r *http.Request) {
-	payload, _ := json.Marshal(haproxy.GetTemplateData(state.Config, state.Zookeeper))
+        templateData, _ := haproxy.GetTemplateData(state.Config, state.Zookeeper)
```

Comment:
I'm not sure what the best way to handle this failure case is so i left it the same for now.  (It looks like the page loads without a list of apps).